docs/lasagne.rst

```diff
@@ -54,9 +54,12 @@ API
 
 .. automodule:: nolearn.lasagne
 
-  .. autoclass:: NeuralNet
+  .. autoclass:: NeuralNet(self,layers,**kwargs)
```

Comment:
What's wrong with auto-grabbing the function signature?  Or am I misunderstanding why you're listing the arguments explicitly?